sources/europe/de/Brandenburg-DOP20c.geojson

```diff
@@ -0,0 +1,1564 @@
+{
+  "type": "Feature",
+  "properties": {
+    "id": "Brandenburg-DOP20c",
+    "name": "Brandenburg GeoBasis-DE/LGB (2020-2022) / DOP20c",
+    "type": "wms",
+    "category": "photo",
+    "url": "https://isk.geobasis-bb.de/mapproxy/dop20c/service/wms?FORMAT=image/png&TRANSPARENT=TRUE&VERSION=1.3.0&SERVICE=WMS&REQUEST=GetMap&LAYERS=bebb_dop20c&STYLES=&CRS={proj}&WIDTH={width}&HEIGHT={height}&BBOX={bbox}",
+    "best": true,
+    "start_date": "2020",
+    "end_date": "2022",
+    "country_code": "DE",
+    "description": "Digitale Orthophotos f\u00fcr das Brandenburg mit einer Bodenaufl\u00f6sung von 0,20 m. Regionen haben unterschiedlich alte Daten, vgl. https://data.geobasis-bb.de/geobasis/information/aktualitaeten/bb_dop_aktualitaet.pdf.",
+    "available_projections": [
+      "EPSG:25833",
+      "EPSG:25832",
+      "EPSG:4326",
+      "EPSG:4258",
+      "EPSG:3034",
+      "EPSG:3035",
+      "EPSG:3045",
+      "EPSG:3857",
+      "EPSG:4839",
+      "EPSG:5650"
+    ],
+    "attribution": {
+      "text": "GeoBasis-DE/LGB 2020 / Digitale farbige Orthophotos aktuell; Geoportal Berlin, dl-de/by-2-0",
```

Comment:
👀  I think we should change this to…

```suggestion
      "text": "GeoBasis-DE/LGB / BB-BE DOP20c, dl-de/by-2-0; Geoportal Berlin / DOP20, dl-de/by-2-0",
```

Why / Context: 

- The official source says the usage requires naming the source like this
   > © GeoBasis-DE/LGB, dl-de/by-2-0; © Geoportal Berlin, dl-de/by-2-0 (Daten geändert)
   https://geobroker.geobasis-bb.de/gbss.php?MODE=GetProductPreview&PRODUCTID=7a503f0f-db46-4772-80e3-b27733fd7acd
- Our official Wiki agreement states that the source tag should follow a different format
  > GeoBasis-DE/LGB (Jahr)/ [Titel des Datensatzes] 
  https://wiki.openstreetmap.org/wiki/Brandenburg/Geoportal#Links and https://wiki.openstreetmap.org/wiki/Brandenburg/Geoportal#Changeset-Attributierung
- We have a composite data source which has (a) data from different years for Brandenburg and (b) data for Berlin included and we do not know which data for Berlin

How to we combine this…

- Adding the year like the wiki suggests does not make sense for a source with different dates, so lets skip this.
- The wiki says to reference the data source, which would officially be "Digitale Orthophotos 20cm Bodenauflösung Farbe Brandenburg mit Berlin (WMS)" which is too long. However, the wiki examples show the usage of short versions. On the product page we can see **"BB-BE DOP20c",** so that is what I would suggest using.
- For Berlin the attribution depends on the (in this case unknown) data. [Example 1](https://github.com/osmlab/editor-layer-index/blob/gh-pages/sources/europe/de/Berlinaerialphotograph2021.geojson?short_path=2f58430#L13), [Example 2](https://github.com/osmlab/editor-layer-index/blob/gh-pages/sources/europe/de/Berlinaerialphotograph2022.geojson?short_path=8913933#L14). I suggest we reduce it to the short version of what all have in common, which is **"DOP20"**
- The official source tells us to add "**dl-de/by-2-0** (Daten geändert)" but the Wiki never mentions it for the source. But the wiki talks about the source-tag on changesets, not what to display, so we should add it. However, I think we can skip the "Daten geändert" part since I do not see any relevance for this use case.

---

Btw, that is what JOSM does (https://josm.openstreetmap.de/wiki/Maps/Germany)

>         <attribution-text>GeoBasis-DE/LGB 2022 / Digitale farbige Orthophotos aktuell</attribution-text>

I thinks the solution above is more in line with the intent of the source provider.

Reply:
Change applied in https://github.com/osmlab/editor-layer-index/pull/2109